lib/utils/array.js

```diff
@@ -0,0 +1,90 @@
+// Production steps of ECMA-262, Edition 6, 22.1.2.1
+// es6-ified by @alexsasharegan
+if (!Array.from) {
+    Array.from = (function() {
```

Comment:
Do we want to actually polyfill `Array.from`, or just assign `from` the code and export `from`?